english/explore-outside/journey-to-flaming-tower/walk/portal/wishes/eagle/eagle.md

```diff
@@ -0,0 +1,3 @@
+With the wave of his hands, the little fellow summons an eagle! You both jump on the back of it and 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.